docs/extend/modular-resources/create/_index.md

```diff
@@ -162,35 +168,13 @@ class MyBase(Base, Reconfigurable):
     # is_moving: check if either motor on the base is moving with motors' is_powered
     async def is_moving(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs) -> bool:
         return self.left.is_powered(extra=extra, timeout=timeout)[0] or self.right.is_powered(extra=extra, timeout=timeout)[0]
-```
 
-</details>
-<br>
-<file>__init__.py</file> registers the mybase custom model and API helper functions with the SDK.
-
-<details>
-  <summary>Click to view sample code from <file>__init__.py</file></summary>
-
-``` python {class="line-numbers linkable-line-numbers"}
-"""
-This file registers the MyBase model with the Python SDK.
-"""
-
-from viam.components.motor import *
-from viam.components.base import Base
-from viam.resource.registry import Registry, ResourceCreatorRegistration
-
-from .my_base import MyBase
-
-Registry.register_resource_creator(Base.SUBTYPE, MyBase.MODEL, ResourceCreatorRegistration(MyBase.new_base, MyBase.validate_config))
 ```
 
-</details>
-
 {{% /tab %}}
 {{% tab name="Go"%}}
 
-<file>mybase.go</file> implements a custom model of the base component built-in resource, "mybase", and registers the new model and API helper functions with the SDK.
+<file>mybase.go</file> implements a custom model of the base component built-in resource, "mybase", and registers the new model and API helper functions with the Go Client SDK.
```

Comment:
same edit as above. A module is also running as a server rather than a client, so Go SDK would be more appropriate
```suggestion
<file>mybase.go</file> implements "mybase", a custom model of the base component,  and registers the new model and API helper functions with the Go SDK.
```

Reply:
Cool, LGTM!

Reply:
done-- sorry for comment spam but i lose track of docs feedback sometimes if I don't comment when I finish or actually commit things 😵‍💫